code/datums/mind/mind.dm

```diff
@@ -90,10 +87,7 @@
 /datum/mind/proc/transfer_to(mob/living/new_character)
 	if(!istype(new_character))
 		to_world_log("## DEBUG: transfer_to(): Some idiot has tried to transfer_to() a non mob/living mob. Please inform Carn")
-	if(current)					//remove ourself from our old body's mind variable
-		if(changeling)
-			current.remove_changeling_powers()
-			current.verbs -= /datum/changeling/proc/EvolutionMenu
+	if(current?.mind == src) //remove ourself from our old body's mind variable
```

Comment:
Is it fine to gate `SSnano.user_transferred(current, new_character)` behind this instead of just behind `if(current)`?